README.md

```diff
@@ -16,4 +16,79 @@ New serialization approaches should be in their own directories with an appropri
 
 We will do a minimum review of new proposals before merging.
 
-Suggestions to existing serialization approaches sill be reviewed by the contributors of the original serializations approach before merging.
+Suggestions to existing serialization approaches will be reviewed by the contributors of the original serializations approach before merging.
+
+## Operating Principles
```

Comment:
maybe also highlight, that just random and useful examples are accepted? That this is described rigid framework is one way, but one can also add arbitrary other examples if they are helpful to play around or to make a point.

Reply:
I think that's covered in "as many or as few examples", but can add text that you suggest.